Components/LandingNav.tsx

```diff
@@ -24,28 +24,66 @@ const LandingNav = () => {
         </div>
         {/* Nav Links */}
         <div className="flex flex-row items-center gap-8">
-          <Link className={navLinkStyle} href="#">
+          <Link
+            spy={true}
+            smooth={true}
+            offset={-70}
+            duration={500}
+            className={navLinkStyle}
+            activeClass="active"
+            to="home"
+          >
```

Comment:
Instead of repeating the same thing repeatedly, we can create a common component for `Link` and keep the same values as default. That would be the right approach.